Formula/polynote.rb

```diff
@@ -0,0 +1,47 @@
+class Polynote < Formula
+  desc "The polyglot notebook with first-class Scala support"
+  homepage "https://polynote.org/"
+  url "https://github.com/polynote/polynote/releases/download/0.3.0/polynote-dist.tar.gz"
+  version "0.3.0"
+  sha256 "b0ad435bd93b36ffcf07da2de5ab1755c02953d902892c5e4a2948144ca5b92f"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
```

Comment:
Please depend on `openjdk`

Reply:
I changed it to 
```
depends_on "openjdk"
```
following the model used by other formulae (`tomcat`, `maven` and `gradle`, for instance). However, `brew audit --new-formula polynote` yields:
```
* Dependency 'openjdk' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.
```
How should I proceed?

Reply:
@SMillerDev 
I updated the formula to depend on `openjdk`. I still see the audit issue mentioned above. Running just `brew audit` or `brew audit --strict` shows no issues, though.